sources/common/infoschema.go

```diff
@@ -41,7 +41,7 @@ type InfoSchema interface {
 	GetIndexes(conv *internal.Conv, table SchemaAndName, colNameIdMp map[string]string) ([]schema.Index, error)
 	ProcessData(conv *internal.Conv, tableId string, srcSchema schema.Table, spCols []string, spSchema ddl.CreateTable, additionalAttributes internal.AdditionalDataAttributes) error
 	StartChangeDataCapture(ctx context.Context, conv *internal.Conv) (map[string]interface{}, error)
-	StartStreamingMigration(ctx context.Context, client *sp.Client, conv *internal.Conv, streamInfo map[string]interface{}) error
+	StartStreamingMigration(ctx context.Context, client *sp.Client, conv *internal.Conv, streamInfo map[string]interface{}) (string, string, error)
```

Comment:
[Optional] `jobId` and `gCloudCmd` are Dataflow specific return values, while `StartStreamingMigration` does a bunch of other things. I suggest we define the return type of `StartStreamingMigration`  accordingly - something like `(DataflowOutputs dataflowOutputs)` maybe?

Reply:
Done.